Dockerfile

```diff
@@ -1,12 +1,22 @@
-FROM golang:1.10
+FROM golang:1.10.4-alpine as builder
 
-COPY . /go/src/github.com/alibaba/Dragonfly
+WORKDIR /go/src/github.com/dragonflyoss/Dragonfly
+COPY . /go/src/github.com/dragonflyoss/Dragonfly
 
-WORKDIR /go/src/github.com/alibaba/Dragonfly/build
+RUN apk --no-cache add bash make gcc libc-dev git
 
-RUN ./build.sh client && cd client && make install
+RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -o /dfclient/dfdaemon cmd/dfdaemon/main.go
+RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -o /dfclient/dfget cmd/dfget/main.go
+
+FROM alpine
+
+RUN apk --no-cache add ca-certificates bash
+
+WORKDIR /dfclient
+
+COPY --from=builder /dfclient /dfclient
 
 # dfdaemon will listen 65001 in dafault.
 EXPOSE 65001
 
-ENTRYPOINT ["dfdaemon"]
+ENTRYPOINT [ "./df-daemon", "--registry", "https://index.docker.io" ]

```

Comment:
If you have set the workdir, please avoid to use `./`. @Starnop

Reply:
Linux will search all paths in the ENV $PATH which the current directory is not in , so we need it and I use the absolute path here instead.